layouts/partials/fragments/member.html

```diff
@@ -234,30 +237,32 @@ <h6 class="font-italic mb-0">
                       </div>
                     {{- end -}}
                     {{- if $member.icons -}}
-                    <div class="row justify-content-center pt-4">
-                      {{- range $member.icons -}}
-                        <span class="fa-stack fa-2x m-2">
-                          <a href="{{ .url }}">
-                            <i class="fas fa-circle fa-stack-2x
-                              {{- if eq $bg "primary" -}}
-                                {{- printf " text-%s" "dark" -}}
-                              {{- end -}}
-                            "></i>
-                            <i class="{{ .icon }} fa-stack-1x fa-inverse"></i>
-                          </a>
-                        </span>
-                      {{- end -}}
-                    </div>
-                    {{- end -}}
-                    {{- with .Content -}}
-                      <div class="pt-3">
-                        <div class="description">
-                          {{- . | markdownify -}}
+                      <div class="col-12 text-center">
```

Comment:
This part of the template is only needed if there are member icons. If there are none then there is a padding introduced by `py-2` on the next element and some weird styles that make `col-12`. IMO only render this part if there are `$member.icons`

Reply:
Just went through the code. Isn't that already the case?
This is around the mentioned div `{{- if $member.icons -}}.

Reply:
Oh yeah you're right. Sorry. I mistook line 251 as `added` instead of `removed`. Monitor color settings :D